wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractEnvironment.java

```diff
@@ -235,8 +233,6 @@ public int getActionStep() {
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.